plugins/connection/aws_ssm.py

```diff
@@ -460,43 +463,45 @@ def _wrap_command(self, cmd, sudoable, mark_start, mark_end):
         if self.is_windows:
             if not cmd.startswith(" ".join(_common_args) + " -EncodedCommand"):
                 cmd = self._shell._encode_script(cmd, preserve_rc=True)
-            cmd = cmd + "; echo " + mark_start + "\necho " + mark_end + "\n"
+            cmd = cmd + "; echo " + mark_start + "; echo " + mark_end + "\n"
```

Comment:
@richardsonky can you rebase with `main` branch and see if this works also in Ubuntu 22.04 dash? Because this PR would include https://github.com/ansible-collections/community.aws/pull/1597/files#diff-8000d7fb9262e11565b61882567d729fe0958cd9b1a0105683d5bbe0e5b4e585R466